### DISCUSSION TITLE
Beginner's question

### DISCUSSION THREAD

**Discussion by Vince56190** [ezylang/EvalEx#439] [Q&A] [answered]
Beginner's question, I would like the displayed value not to exceed a defined value ( 3543). is this the right formula: ((a*0.2766)-190),IF (a>3543,"3543,a)
Thx

**Comment by RobertZenz** [comment, marked as answer]
Using `min` would be easier:

    min(3543, ((a*0.2766)-190))

`min` will select the *smaller* value, so either the result of the calculation, or `3543` if the result is larger.

**Reply by Vince56190** [reply]
Thanx  n I will try that.

**Comment by mrunali0204** [comment]
Hi @Vince56190!
To make sure the result of ((a * 0.2766) - 190) never exceeds 3543, the best approach is to use the min() function like this:
min(3543, ((a * 0.2766) - 190))
This compares the result of the formula and 3543
If the result is greater than 3543, it returns 3543
If it’s less than 3543, it returns the result as-is
It’s clean, simple, and works in most programming languages or spreadsheets that support min().
